app/views/volunteers/_form.html.slim

```diff
@@ -0,0 +1,115 @@
+= simple_form_for(@volunteer) do |f|
+  - if f.error_notification.present?
+    .row
+      .col-xs-12
+        = f.error_notification
+
+  .row
+    .col-xs-12.col-md-6
+        = f.input :first_name
+
+    .col-xs-12.col-md-6
+        = f.input :last_name
+
+  .row
+    .col-xs-12.col-md-6
+      = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
```

Comment:
I did quite some research but I didn't find anything useful. Even at the simple_form documentation examples, they have it as we currently do. If you, or @casaper and @nuritnt have a suggestion or hint about this, please let me know.